content/en/docs/concepts/storage/volumes.md

```diff
@@ -121,10 +121,10 @@ beta features must be enabled.
 
 #### AWS EBS CSI migration complete
 
-{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
```

Comment:
why change this line?
A related question is why do we rename the gate?
This change will confuse a lot of users.

Reply:
This is to decouple unregistering a plugin from CSI migration. Previously if you want to disable an intree plugin. There is no flag that can do it. The only way is to enable CSI migration and enable this Complete flag.

Now with the new flag. You can set it directly to disable an intree plugin without enabling CSI migration.

Reply:
Done